specifications/default-tracks.org

```diff
@@ -0,0 +1,87 @@
+#+TITLE: Default Tracks
+#+AUTHOR: Sergio Schvezov <sergio.schvezov@canonical.com>
+#+DATE: [2020-07-20]
+
+* Scope
+Allow for operations relevant to default tracks using the Snapcraft command
+line.
+
+The new commands introduced shall allow for viewing tracks and their
+corresponding status as well as setting a track as a default one.
+
+* User Experience
+** Viewing Tracks
+A new command shall be introduced, =snapcraft list-tracks= with an alias of
+=tracks=. The command shall display the list of available tracks with potential
+status values of =active=, =hidden= and =closed=. When a default track is set, the
+status shall be set to =default= which will implicitly mean it is =active=.
+ 
+The command shall be defined as:
+
+#+BEGIN_SRC sh
+snapcraft track|list-tracks <snap-name>
+#+END_SRC
+
+With that criteria, this is how the output shall display for a given snap:
+#+BEGIN_SRC sh
+  $ snapcraft list-tracks node
+  Track    Status
+  17       active
+  16       default
+  15       active
+  14       hidden
+  13       closed
+#+END_SRC
+
+#+BEGIN_QUOTE
+The =hidden= and =closed= Status values are shown for illustration purposes, only
+active and default shall be implemented as part of this specification.
+#+END_QUOTE
+
+** Setting a default track
```

Comment:
is it worth mentioning how to "remove " a default track by pointing it to latest?

Reply:
I thought it was agreed that this was not a "removal", please correct me if I am wrong.

Reply:
no ideed it's not a removal.

I was just wondering if you were capturing also those details of the spec in here: ~"To revert back your change to the behaviour, the user has to set the default track to latest." 

So basically there is no removal, just setting the default track back the latest branch.